### DISCUSSION TITLE
Dynamic Prompts feature has disappeared, can anyone help?

### DISCUSSION THREAD

**Discussion by rpillala** [adieyal/sd-dynamic-prompts#100] [Q&A] [answered]
Can anyone advise me?  I updated extensions using the webui (check for updates --> apply and restart UI) and now the Dynamic Prompts section on txt2img is gone, and the extension doesn't seem to be functional.  My prompt is sexy vampire, \_\_people/gem\_\_, where I have a text file called "gem.txt" containing different gems like quartz, ruby, etc.  After generation the parameters show "\_\_people/gem\_\_" rather than a term from the file.  When I've used this successfully in the past, the generation parameters would show a term from the file instead.  And, while typing the double underscores, the autocomplete extension does suggest files and terms including the ones that ship with the extension, as well as my custom files.  That is, I see a dropdown list with people/gem : amber, people/gem : amethyst, etc.  So, it is aware of the files and terms, but just isn't using them in generation. 

At first I thought the Dynamic Prompts options might be in Settings now, but I don't see anything.

**Comment by adieyal** [comment, marked as answer]
It's working for me - can you check to make sure the dynamic prompts checkbox is enabled?
![Screenshot from 2022-12-11 11-40-02](https://user-images.githubusercontent.com/420786/206896565-d83eec75-4958-4d0f-9711-5d15d9f53318.png)

**Reply by rpillala** [reply]
When I posted, that section of the txt2img tab was gone, and so there was no checkbox for dynamic prompts.  Now it's back and I'm not sure how it happened.  Thanks for checking!
